### ISSUE TITLE
No module named 'dns'

### ISSUE THREAD

**Issue by niftylettuce** [ValiMail/authentication-headers#8]
```sh
Traceback (most recent call last):␊
      File "/Users/test/Projects/authheaders/scripts/authenticate-message.py", line 3, in <module>␊
        import authheaders␊
      File "/usr/local/lib/python3.8/site-packages/authheaders/__init__.py", line 23, in <module>␊
        from authheaders.dmarc_lookup import dns_query, receiver_record, get_org_domain␊
      File "/usr/local/lib/python3.8/site-packages/authheaders/dmarc_lookup.py", line 28, in <module>␊
        from dns.resolver import (query, NXDOMAIN, NoAnswer, NoNameservers)␊
    ModuleNotFoundError: No module named 'dns'
```

**Comment by niftylettuce** [comment]
Python 3.8.3

```sh
❯ pip3 install dnspython publicsuffix2 ipaddress authres dkimpy
Requirement already satisfied: dnspython in /usr/local/lib/python3.8/site-packages (2.0.0)
Requirement already satisfied: publicsuffix2 in /usr/local/lib/python3.8/site-packages (2.20191221)
Requirement already satisfied: ipaddress in /usr/local/lib/python3.8/site-packages (1.0.23)
Requirement already satisfied: authres in /usr/local/lib/python3.8/site-packages (1.2.0)
Requirement already satisfied: dkimpy in /usr/local/lib/python3.8/site-packages (1.0.4)
Requirement already satisfied: Py3DNS in /usr/local/lib/python3.8/site-packages (from dkimpy) (3.2.1)
```

**Comment by niftylettuce** [comment]
I solved this by building from source per this answer <https://stackoverflow.com/a/21642297>.

```sh
git clone https://github.com/rthalley/dnspython
cd dnspython
python3 setup.py install
```

**Comment by niftylettuce** [comment]
Using `dnspython==2.1.0.dev1` if anyone interested.

**Comment by niftylettuce** [comment]
Actually, the fix is to use `dnspython==1.16.0`

**Comment by kitterma** [comment]
As long as you upgrade dkimpy to 1.0.5, that should be fine without pinning the dnspython version.
